docs/lang/const.md

```diff
@@ -1,25 +1,25 @@
-C++ 定义了一套完善的只读量定义方法，被常量修饰符 `const` 修饰的对象或类型都是只读量，只读量的内存存储与一般变量没有任何区别，但是编译器会在编译期进行冲突检查，避免对只读量的修改。因此合理使用 `const` 修饰符可以增加代码健壮性。
+C++ defines a complete set of methods for defining read-only variables. Objects or types modified by the constant modifier `const` are all read-only. The memory storage of read-only variables is no different from general variables, but the compiler will compile periodically check for conflicts to avoid modification of read-only variables. Therefore, reasonable use of the `const` modifier can increase code robustness.
```

Comment:
```suggestion
C++ defines a complete set of methods for defining read-only variables. Objects or types modified by the constant modifier `const` are all read-only. The memory storage of read-only variables is of no difference from general variables defining, but the compiler will check for conflicts to avoid modification of read-only variables during the compilation phase. Therefore, reasonable use of the `const` modifier can increase code robustness.
```